### DISCUSSION TITLE
2005/12/08/%E6%8B%94%E6%8E%89%E4%BA%86%E6%A8%AA%E7%9D%80%E9%95%BF%E7%9A%84%E6%99%BA%E9%BD%BF

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#66] [Announcements]
# 2005/12/08/%E6%8B%94%E6%8E%89%E4%BA%86%E6%A8%AA%E7%9D%80%E9%95%BF%E7%9A%84%E6%99%BA%E9%BD%BF

一年前就打算要拔掉这颗智齿了。当时去看牙医的时候，医生发现我右下方本来该长智齿的地方只能隐约看到牙龈下面包着一颗牙齿。但现在他还没长出来，那肯定是长得不对了。于是去拍了X光片，发现那颗智齿原来是横着长得，顶在它前面的磨牙和下颌骨之间。

https://blog.qizhen.xyz/2005/12/08/%E6%8B%94%E6%8E%89%E4%BA%86%E6%A8%AA%E7%9D%80%E9%95%BF%E7%9A%84%E6%99%BA%E9%BD%BF

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Zhen
2009年07月22日 22:42 编辑
现在的我就像当时的你一样~痛苦~

回复
泽丽
2008年09月22日 13:38 编辑
昨天看了你的日志，今天早上终于狠下心去拔了。
去医院之前的心情是最恐怖的，知道得太详细，纒着咨询的医生问是不是一定要敲碎才能拔出来，她回我说既然来了就别问太多。
嘿嘿！上了麻药之后闭上眼睛就任人宰割。虽然不知道疼，眼泪还是飚了出来。
 
 

回复
Unknown
2005年12月27日 20:50 编辑
还不快谢谢我 哼！
